concepts/05 Widgets/TabPanel/15 Control the Behavior.md

```diff
@@ -1,44 +1,481 @@
 An end user can switch the views by swiping them or selecting tabs. Note that tab navigation is always enabled, while swiping is active only on touch devices by default. To enable it on all types of devices, assign **true** to the [swipeEnabled](/api-reference/10%20UI%20Widgets/dxTabPanel/1%20Configuration/swipeEnabled.md '/Documentation/ApiReference/UI_Widgets/dxTabPanel/Configuration/#swipeEnabled') option. Otherwise, assign **false**.
 
+---
+
+##### jQuery
+
     <!--JavaScript-->
-    $(function() {
-        $("#tabPanelContainer").dxTabPanel({
-            items: tabPanelItems,
+    $(function () {
+        $('#tabPanelContainer').dxTabPanel({
+            items: [{
```

Comment:
The data source doesn't add anything to the example. Let's remove it from all examples in this article.